README.md

```diff
@@ -161,6 +161,25 @@ To use the default theme, please do the following:
 
    Source: [src/core/styles/common/selectors](src/core/styles/common/selectors)
 
+1. CSS & SCSS Variables
+
+E.g., CSS: `--sds-color-beta-400`, `--sds-spaces-m`; SCSS: `$sds-color-beta-400`, `$sds-spaces-m`
+
+Source: [src/common/styles-dictionary](src/common/styles-dictionary)
+
+These stylesheets can be imported into the index.js file of your application to make use of czifui's standard styles in projects that use SCSS or CSS modules to style the front end:
+
+```js
+// example using create-react-app structure
+// index.js
+
+import React from "react";
+import ReactDOM from "react-dom";
+import "czifui/dist/variables.css";
```

Comment:
I was thinking about adding `scss` example too, especially since the export file has `_` prefix!

```suggestion
// If you need css
import "czifui/dist/variables.css";
// If you need scss
import "czifui/dist/_variables.scss";
```